openedx/features/colaraz_features/helpers.py

```diff
@@ -0,0 +1,135 @@
+"""
+Helper functions for colaraz app.
+"""
+from collections import namedtuple
+
+from django.contrib.sites.models import Site
+
+from organizations.models import Organization
+
+from openedx.core.djangoapps.theming.models import SiteTheme
+from openedx.core.djangoapps.site_configuration.models import SiteConfiguration
+
+
+# Tuple containing information for lms and studio
+# This named tuple can be used for sites, site themes, organizations etc.
+Pair = namedtuple('Pair', 'lms studio')
+
+
+def remove_duplicates(items):
+    """
+    Remove duplicate items from the argument list.
+
+    The function will not work if the items in the provided list are not hashable.
+
+    Arguments:
+        (list): List of hashable items from which to remove duplicates.
+
+    Returns:
+        (list): List of items containing only unique items.
+    """
+    return list(set(items))
+
+
+def create_sites(domain, name):
+    """
+    Create sites for lms and studio with domain and name provided in the arguments.
+
+    LMS Domain will be formed by prepending `courses` to the passed in domain. And Studio domain will be
+    formed by prepending `studio` to the passed in domain.
+
+    Arguments:
+        domain (str): Domain to identify the site. e.g. example.com, test.example.com etc.
+        name (str): Human readable name for the site.
+
+    Returns:
+        (Pair<Site, Site>): named tuple containing lms, studio sites.
+    """
+    sites = (
+        Site.objects.create(
+            domain='{}.{}'.format(prefix, domain),
+            name=name
```

Comment:
```suggestion
            name='{}.{}'.format(prefix, name)
```
To distinguish between LMS and Studio sites.